cypress/integration/finance/ledgers/ledgers.create.spec.js

```diff
@@ -0,0 +1,25 @@
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+import TopMenu from '../../../support/fragments/topMenu';
+
+describe('ui-finance: Ledger list search and filters', () => {
+  const timestamp = (new Date()).getTime();
+
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'))
+      .then(() => {
+        cy.visit(TopMenu.ledgerPath);
+      });
+  });
+
+  it('C4053 should create new ledger if mandatory fields are filled', function () {
+    Ledgers.createDefaultLedger();
+    Ledgers.deleteLedgerViaActions();
+  });
+
+  it('C4053 should not create new ledger if mandatory fields are not filled', function () {
```

Comment:
think with PO to split this test in testrail to 2 tests. Not good idea to have many-to-many relations between cypress and testrail tests